include/libkrun.h

```diff
@@ -243,6 +243,19 @@ int32_t krun_set_snd_device(uint32_t ctx_id, bool enable);
  */
 int32_t krun_set_rlimits(uint32_t ctx_id, const char *const rlimits[]);
 
+/**
+ * Sets the SMBIOS OEM Strings.
+ *
+ * Arguments:
+ *  "ctx_id"      - the configuration context ID.
+ *  "oem_strings" - an array of string pointers.
+ *  "count"       - number of elements in the array.
+ *
+ * Returns:
+ *  Zero on success or a negative error number on failure.
+ */
+int32_t krun_set_smbios_oem_strings(uint32_t ctx_id, const char *const oem_strings[], char count);
```

Comment:
It's a bit weird passing `count` as `char`. And, while I agree this strategy is better than using `from_raw_parts` with `MAX_ARGS`, please use it and drop `count` for consistency with other functions in the API. We'll change all of them at once in v2.0.

Reply:
Ok, should I explicitly ask for a NULL ptr terminator of the array?, like:

```
/**
 * Sets the SMBIOS OEM Strings.
 *
 * Arguments:
 *  "ctx_id"      - the configuration context ID.
 *  "oem_strings" - an array of string pointers. Is terminated with an additional NULL pointer.
 *
 * Returns:
 *  Zero on success or a negative error number on failure.
 */
int32_t krun_set_smbios_oem_strings(uint32_t ctx_id, const char *const oem_strings[]);

```

I used `char` (btw it should be `unsigned char`) because the SMBIOS use a `u8` to define the number of strings, and I think it's better to have these restriction in the type system than as a runtime check.

Reply:
I just tested it more, and I got a segfault, we need to explicitly ask for a NULL terminator (but the from_raw_part is still UB)

Reply:
I just realized that `inttypes.h` is included... instead of `unsigned char` I could have use `uint8_t`